datalake/src/main/java/com/sequenceiq/datalake/flow/dr/backup/DatalakeDatabaseBackupActions.java

```diff
@@ -159,7 +189,7 @@ protected SdxContext createFlowContext(FlowParameters flowParameters, StateConte
             @Override
             protected void doExecute(SdxContext context, DatalakeDatabaseBackupFailedEvent payload, Map<Object, Object> variables) {
                 Exception exception = payload.getException();
-                LOGGER.error("Datalake database backup failed for datalake with id: {}", payload.getResourceId(), exception);
+                LOGGER.error("Datalake backup failed for datalake with id: {}", payload.getResourceId(), exception);
```

Comment:
If this method is `databasebackupFailed`, we can leave the log message the same as it was before, since it is about the failed database backup.

Reply:
addressed.